ntrans_dataprep.py

```diff
@@ -7,8 +7,10 @@
 
 Unzip BNC to the working directory, and rename the folder `BNC`.
 
-NOTE: The BNC contains around 112.000.000 words (or 5.397.000 sentences).
-It can take around 2 hours to run this script.
+NOTE: The BNC contains around 112.000.000 words (or 6.020.000+ sentences).
```

Comment:
All changes to this file seem to be a part of #4. Let's get that merged first so that its changes won't appear here.

Reply:
Sure